src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -15,6 +15,14 @@
 				disable-tooltip />
 		</template>
 		<h4 class="absence-reminder__caption">{{ userAbsenceCaption }}</h4>
+		<div v-if="userAbsence.replacementUserId" class="absence-reminder__replacement">
+			<!-- TRANSLATORS An acting person during the period of absence of the main contact -->
+			<p>{{ t('spreed','Replacement: ') }}</p>
+			<NcUserBubble class="absence-reminder__replacement__bubble"
+				:display-name="userAbsence.replacementUserDisplayName"
+				:user-id="userAbsence.replacementUserId"
+				@click="openConversationWithReplacementUser" />
+		</div>
```

Comment:
There is no indication on UI that's clickable, I suggest to add `cursor : pointer` and a label like `chat with {user}`

Reply:
I think that should be fixed upstream

Reply:
Made a PR for cursor, will be fixed eventually: https://github.com/nextcloud-libraries/nextcloud-vue/pull/5708
But `title` attribute should be added at Talk app level. We could use something of existing translations, maybe?
- Private conversation
- Join conversation
- Open conversation
- Go to conversation